Fix:

error: implicit declaration of function 'close' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'malloc' with type 'void *(unsigned long)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'memset' with type 'void *(void *, int, unsigned long)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'realloc' with type 'void *(void *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strcpy' with type 'char *(char *, const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2003-04-10 18:15:59.000000000 -0500
+++ configure	2024-07-25 03:20:27.000000000 -0500
@@ -3229,6 +3229,7 @@
 #line $LINENO "configure"
 #include "confdefs.h"
 #include <ctype.h>
+#include <stdlib.h>
 #if ((' ' & 0x0FF) == 0x020)
 # define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
 # define TOUPPER(c) (ISLOWER(c) ? 'A' + ((c) - 'a') : (c))
--- src/HighScoreClient.c.orig	2003-04-10 18:15:59.000000000 -0500
+++ src/HighScoreClient.c	2024-07-25 03:23:29.000000000 -0500
@@ -1,3 +1,6 @@
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
 #include <stdio.h>
 #include <netdb.h>
 #include <term.h>
